modules/gis/docs/api-reference/geojson-to-binary.md

```diff
@@ -19,51 +19,92 @@ const geoJSONfeatures = await load(url, MVTLoader, loaderOptions);
+    // Indices within positions of the start of each individual LineString
+    pathIndices: {value: Uint16Array || Uint32Array, size: 1},
     // Array of original feature indexes by vertex
-    objectIds: {value: Uint32Array, size: 1},
+    globalFeatureIds: {value: Uint16Array || Uint32Array, size: 1},
+    // Array of LineString feature indexes by vertex
+    featureIds: {value: Uint16Array || Uint32Array, size: 1},
+    // Object with accessor objects for numeric properties
+    // Numeric properties are sized to have one value per vertex
+    numericProps: {
+        numericProperty1: {value: Float32Array, size: 1}
+    }
+    // Array of objects with non-numeric properties from LineString geometries
+    properties: [{LineStringFeatureProperties}]
   },
   polygons: {
+    // Array of x, y or x, y, z positions
+    positions: {value: PositionDataType, size: coordLength},
     // Indices within positions of the start of each complex Polygon
-    polygonIndices: {value: Uint32Array, size: 1},
+    polygonIndices: {value: Uint16Array || Uint32Array, size: 1},
     // Indices within positions of the start of each primitive Polygon/ring
-    primitivePolygonIndices: {value: Uint32Array, size: 1},
-    // Array of x, y or x, y, z positions
-    positions: {value: Float32Array, size: coordLength},
+    primitivePolygonIndices: {value: Uint16Array || Uint32Array, size: 1},
     // Array of original feature indexes by vertex
-    objectIds: {value: Uint32Array, size: 1},
+    globalFeatureIds: {value: Uint16Array || Uint32Array, size: 1},
+    // Array of Polygon feature indexes by vertex
+    featureIds: {value: Uint16Array || Uint32Array, size: 1},
+    // Object with accessor objects for numeric properties
+    // Numeric properties are sized to have one value per vertex
+    numericProps: {
+        numericProperty1: {value: Float32Array, size: 1}
+    }
+    // Array of objects with non-numeric properties from Polygon geometries
+    properties: [{PolygonFeatureProperties}]
   }
 }
 ```
 
 ## Options
 
-| Option           | Type                             | Default        | Description                         |
-| ---------------- | -------------------------------- | -------------- | ----------------------------------- |
-| PositionDataType | `Float32Array` or `Float64Array` | `Float32Array` | Data type used for positions arrays |
+| Option           | Type     | Default           | Description                                                                                                                                             |
+| ---------------- | -------- | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| PositionDataType | `class`  | `Float32Array`    | Data type used for positions arrays.                                                                                                                    |
```

Comment:
This reminds me: It is best if the option values are representable in JSON, i.e. numbers, strings, etc. So you could support `"float32"` and `"float64"`, or `positionPrecision: 32 | 64`

The reason is that loaders.gl is integrated with deck.gl, and deck.gl exposes loader options in its API, and for instance pydeck and non-JS language bindings go through the deck.gl JSON API intermediary.